main.nf

```diff
@@ -86,14 +87,14 @@ include { process_split_reads; process_split_reads_mid; process_split_reads_long
 /*
  * Create the header for the output VCF
  */
-process buildHeader {
+process convertAndAddHeaderToVCF {
```

Comment:
what if we change the process name to something like `generateRemappedVCF` and add to the description that we are coverting to VCF and merging the header with the remapped variants?

Reply:
I've address this comment in subsequent PR [EVA-2451 - allow novel ref allele #29
](https://github.com/EBIvariation/variant-remapping/pull/29)